src/main/java/net/dv8tion/jda/api/entities/Member.java

```diff
@@ -392,20 +393,36 @@ default ImageProxy getEffectiveAvatar()
      * <p>This is determined by the color of the highest role assigned to them that does not have the default color.
      * <br>If all roles have default color, this returns null.
      *
+     * @deprecated It's better to use {@link RoleColors#getPrimaryColor() getColors().getPrimaryColor()}.
```

Comment:
Move deprecation message below and align it with other tag's content (such as `@return`)